README.md

```diff
@@ -83,6 +84,8 @@ sudo apt-get update
 sudo apt-get install git clang-format-5.0 clang-tidy-5.0 clang-5.0 libboost-system-dev libboost-filesystem-dev libboost-test-dev libssl-dev libleveldb-dev libjsoncpp-dev libsnappy-dev cmake libmicrohttpd-dev libjsonrpccpp-dev build-essential pkg-config libevent-dev libminiupnpc-dev ocl-icd-opencl-dev
 ```
 
+If you want to use CUDA GPU, please use `sudo ./scripts/install-cuda-ubuntu1604.sh` to install CUDA library first, you may need to reboot your PC to make the CUDA installation take effect.
+
```

Comment:
It's better to leave CUDA installation for users because it requires some familiarity and experiences with CUDA. It's okay we provide some quick installation (like the script `install-cuda-ubuntu1604.sh`) but I suggest moving it to our wiki from the codebase as (1) GPU mining (as well as CUDA) is not a compulsory requirement; (2) we may edit the installation instructions frequently when we are adding support for more hardware; (3) previously, the intention of having the script is to run it on Travis but we don't need it anymore.

Reply:
Ok, understand, i will provide the nvidia website link instead.